packages/fx-core/src/question/create.ts

```diff
@@ -416,6 +416,8 @@ export class CapabilityOptions {
       ...CapabilityOptions.tabs(),
       ...CapabilityOptions.mes(),
       CapabilityOptions.copilotPluginCli(),
+      //add search me options here to unblock e2e test
+      CapabilityOptions.SearchMe(),
```

Comment:
I find there is SearchMe option in line 387. Can we use that option for e2e test?

Reply:
staticAll() cannot distinguish dotnet and node. You can check L549.